qiskit/pulse/library/pulse.py

```diff
@@ -39,6 +39,10 @@ def id(self) -> int:  # pylint: disable=invalid-name
         """Unique identifier for this pulse."""
         return id(self)
 
+    def is_parameterized(self) -> bool:
```

Comment:
This and the method below should be decorated with `abstractmethod`

Reply:
Fixed in https://github.com/Qiskit/qiskit-terra/pull/5250/commits/9239d66ed6a9f9c3ded453045292092748e3c232